src/gui/macutilities.h

```diff
@@ -0,0 +1,13 @@
+#ifndef MACUTILITIES_H
```

Comment:
I noticed Mike used his github name and went with that but I can change it to my email.